api/PointerEvent.json

```
@@ -569,7 +569,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": false,
+                "version_added": "26.2",
                 "impl_url": "https://webkit.org/b/279540"
```

Removing the bug reference since there is nothing to see anymore

```suggestion
                "version_added": "26.2"
```